core/src/main/java/com/linecorp/armeria/common/stream/FuseableStreamMessage.java

```diff
@@ -252,6 +252,19 @@ public void cancel() {
             };
         }
 
+        /**
+         * Use {{@link #and(MapperFunction)}} instead.
+         */
+        @Override
+        default <V> Function<T, V> andThen(Function<? super R, ? extends V> after) {
+            throw new UnsupportedOperationException();
+        }
+
+        @Override
+        default <V> Function<V, R> compose(Function<? super V, ? extends T> before) {
+            throw new UnsupportedOperationException();
+        }
```

Comment:
Should we supply an exception message for these methods?

Reply:
Yeah. An exception message would help devs debug from unexpected mistakes.